cmd/transactioncmd/transaction_commit.go

```diff
@@ -30,5 +38,63 @@ func commitTx(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
+	tx, err := txutils.LoadFromDisk(inputTxPath)
+	if err != nil {
+		return err
+	}
+
+	network, err := txutils.GetNetwork(tx)
+	if err != nil {
+		return err
+	}
+
+	subnetName := args[0]
+	sc, err := app.LoadSidecar(subnetName)
+	if err != nil {
+		return err
+	}
+	subnetID := sc.Networks[network.String()].SubnetID
+	if subnetID == ids.Empty {
+		return errNoSubnetID
+	}
+
+	subnetAuthKeys, err := txutils.GetAuthSigners(tx, network, subnetID)
+	if err != nil {
+		return err
+	}
+
+	remainingSubnetAuthKeys, err := txutils.GetRemainingSigners(tx, network, subnetID)
+	if err != nil {
+		return err
+	}
+
+	if len(remainingSubnetAuthKeys) != 0 {
+		signedCount := len(subnetAuthKeys) - len(remainingSubnetAuthKeys)
+		ux.Logger.PrintToUser("%d of %d required signatures have been signed.", signedCount, len(subnetAuthKeys))
+		subnetcmd.PrintRemainingToSignMsg(subnetName, remainingSubnetAuthKeys, inputTxPath)
+		return nil
+	}
+
+	// get kc with some random address, to pass wallet creation checks
+	kc := secp256k1fx.NewKeychain()
+	_, err = kc.New()
+	if err != nil {
+		return err
+	}
+
+	deployer := subnet.NewPublicDeployer(app, false, kc, network)
+	txID, err := deployer.Commit(tx)
+	if err != nil {
+		return err
+	}
+
+	if txutils.IsCreateChainTx(tx) {
+		if err := subnetcmd.PrintDeployResults(subnetName, subnetID, txID, true); err != nil {
+			return err
+		}
+		return subnetcmd.UpdateSidecarNetworks(&sc, network, subnetID, txID)
+	}
+	ux.Logger.PrintToUser("Transaction successful, transaction ID: %s", txID)
```

Comment:
Should we say submitted instead of successful?

Reply:
Or do we want to actually wait for the tx to be accepted?

Reply:
wallet already checks that it is accepted